File: Models/Event.cs
using System;

namespace Angular.Models
{
    public class Event
    {
        public int Id { get; set; }

        public string City { get; set; }

        public string Topic { get; set; }

        public int Subscribers { get; set; }

        public string Group { get; set; }

        public DateTime Date { get; set; }
    }
}

File: ClientApp/src/app/models/Event.ts
import { Group } from "./Group";
import { SocialMedia } from "./SocialMedia";
import { Speaker } from "./Speaker";

export class Event {
  constructor() { }

  id: number;
  city: string;
  topic: string;
  subscribers: number;
  phone: string;
  email: string;
  date: Date;
  imageFile: string;
  groups: Group[];
  socialMedias: SocialMedia[];
  speakerEvents: Speaker[];
}
